specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/DeletedAccountGet.json

```diff
@@ -9,16 +9,13 @@
   "responses": {
     "200": {
       "body": {
-        "id": "/subscriptions/{subscription-id}/providers/Microsoft.Storage/locations/eastus/deletedAccounts/sto1125",
+        "key": "sto1125|2020-08-17T03:35:37.4588848Z",
         "name": "sto1125",
-        "type": "Microsoft.Storage/deletedAccounts",
-        "properties": {
-          "storageAccountResourceId": "/subscriptions/{subscription-id}/resourceGroups/sto/providers/Microsoft.Storage/storageAccounts/sto1125",
-          "location": "eastus",
-          "restoreReference": "sto1125|2020-08-17T03:35:37.4588848Z",
-          "creationTime": "2020-08-17T03:35:37.4588848Z",
-          "deletionTime": "2020-08-17T04:41:37.3442475Z"
-        }
+        "location": "eastus",
+        "subscription": "45b60d85-fd72-427a-a708-f994d26e593e",
```

Comment:
these are top level properties, which are known to ARM. Not sure if they will be allowed. Usually we put them "properties".

Reply:
The response provided here is from an actual response on our API. I think reflects the real behavior?